client/app/utils/jsonSchema/operations.ts

```diff
@@ -457,9 +459,7 @@ export const operationUiSchema = {
     "ui:widget": "RadioWidget",
   },
   is_application_lead_external: {
-    "Would you like to add an exemption ID application lead?": {
```

Comment:
Why do we need to remove this?

Reply:
Apologies, I should have put a comment in this PR as it is admittedly not related to this PR but a bug that needed to be fixed. I believe this is a mistake as the field used to be called that and was changed to `is_application_lead_external`. This is basically two fields wrapping each other and as a result the `RadioWidget` wasn't used and it's just incorrectly displayed as a checkbox.

Reply:
I think that solves the first screenshot in this bug ticket then? https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/409

Reply:
Yes that is it, I will remove it from the ticket thanks!